pkgs/development/tools/clang-tools/default.nix

```
@@ -1,4 +1,4 @@
-{ lib, stdenv, llvmPackages }:
+{ lib, stdenv, llvmPackages, cxx ? false }:
```

cxx might collide with a package name one day:

```suggestion
{ lib, stdenv, llvmPackages, enableLibcxx ? false }:
```

---

yeah, good point!